VEHICLE/GetVehiclePlateType.md

```diff
@@ -8,6 +8,15 @@ ns: VEHICLE
 int GET_VEHICLE_PLATE_TYPE(Vehicle vehicle);
 ```
 
+```
```

Comment:
Please remove the code block from the description. ([example template](https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md#example-documentation))